tests/Install-DbaMultiTool.Tests.ps1

```diff
@@ -0,0 +1,131 @@
+            $server.Query("CREATE DATABASE $database")
+
+            $outfile = "DBA-MultiTool-$branch.zip"
+            Invoke-WebRequest -Uri "https://github.com/LowlyDBA/dba-multitool/archive/$branch.zip" -OutFile $outfile
+            if (Test-Path $outfile) {
+                $fullOutfile = (Get-ChildItem $outfile).FullName
+            }
+            $resultsLocalFile = Install-DbaMultiTool -SqlInstance $script:instance3 -Database $database -Branch $branch -LocalFile $fullOutfile -Force
+        }
+        AfterAll {
+            Remove-DbaDatabase -SqlInstance $script:instance3 -Database $database -Confirm:$false
+        }
+
+        It "Installs to specified database: $database" {
+            $resultsLocalFile[0].Database -eq $database | Should -Be $true
+        }
+        It "Shows status of Installed" {
+            $resultsLocalFile[0].Status -eq "Installed" | Should -Be $true
+        }
+        It "Installed sp_doc, sp_helpme, sp_sizeoptimiser, and sp_estindex" {
+            'sp_doc', 'sp_helpme', 'sp_sizeoptimiser', 'sp_estindex' | Should -BeIn $resultsLocalFile.Name
+        }
+        It "Has the correct properties" {
+            $result = $resultsLocalFile[0]
+            $ExpectedProps = 'SqlInstance,InstanceName,ComputerName,Name,Status,Database'.Split(',')
+            ($result.PsObject.Properties.Name | Sort-Object) | Should -Be ($ExpectedProps | Sort-Object)
+        }
+        It "Shows status of Updated" {
+            $resultsLocalFile = Install-DbaMultiTool -SqlInstance $script:instance3 -Database $database
+            $resultsLocalFile[0].Status -eq 'Updated' | Should -Be $true
+        }
+        It "Shows status of Error" {
+            $folder = Join-Path (Get-DbatoolsConfigValue -FullName Path.DbatoolsData) -Child "DBA-MultiTool-$branch"
+            $sqlScript = (Get-ChildItem $folder | Select-Object -First 1).FullName
+            Add-Content $sqlScript (New-Guid).ToString()
+            $result = Install-DbaMultiTool -SqlInstance $script:instance3 -Database $database
+            $result[0].Status -eq "Error" | Should -Be $true
+        }
+    }
+}
+Describe "$CommandName Integration Tests" -Tags "IntegrationTests" {
+    Context "Testing DBA MultiTool installer with unsupported SQL Server version" {
+        BeforeAll {
+            $branch = "master"
+            $database = "dbatoolsci_multitool_$(Get-Random)"
+            $server = Connect-DbaInstance -SqlInstance $script:instance1
+            $server.Query("CREATE DATABASE $database")
+
+            $outfile = "DBA-MultiTool-$branch.zip"
+            Invoke-WebRequest -Uri "https://github.com/LowlyDBA/dba-multitool/archive/$branch.zip" -OutFile $outfile
+            if (Test-Path $outfile) {
+                $fullOutfile = (Get-ChildItem $outfile).FullName
+            }
+            $resultsLocalFile = Install-DbaMultiTool -SqlInstance $script:instance1 -Database $database -Branch $branch -LocalFile $fullOutfile -Force
```

Comment:
This should also be possible by lowering the compatibility level of the database unless your stored procedures validate against the version of the instance instead.

Reply:
The stored procedures do use the Major Version of the instance, not the compatibility level. That is why I was trying to use the 2008 instance for the skip tests.

Reply:
Easiest way to handle that if you want to keep the test is mocking the connect command of our module where the property for the version is returned as 2008 R2.

Reply:
Going to remove that test for the time being as it'd probably require re-writing portions of the actual function and I would like to hold off on that for the time being